src/asm/rgbasm.5

```diff
@@ -812,6 +812,66 @@ first, followed by the one from
 and the one from
 .Ql bar.o
 last.
+.Ss Inline Fragments
+Inline fragments are useful for short blocks of code or data that are only referenced once.
+They are section fragments created by surrounding instructions or directives with
+.Ql [[
+double brackets
+.Ql ]] ,
+without a separate
+.Ic SECTION FRAGMENT
+declaration.
+.Pp
+The content of an inline fragment becomes a section fragment, sharing the same constraints as its parent ROM section.
+Its parent section also becomes a section fragment if it is not one already, and RGBLINK merges them together.
+Inline fragments are appended after their parent section in no particular order.
```

Comment:
I'm not sure...
```asm
SECTION "A", ROM0
    ld hl, [[ nop ]]
PUSHS
SECTION FRAGMENT "A", ROM0
    ret
POPS
    ld de, [[ rrca ]]
```
I think this would currently output
```asm
    ld hl, .lab1
    ld de, .lab2
.lab1
    nop
    ret
    rrca
```

Reply:
On the one hand, that example outputs the same ROM as this one without inline fragments:

```
SECTION FRAGMENT "A", ROM0
    ld hl, Foo
PUSHS
SECTION FRAGMENT "A", ROM0
    ret
POPS
    ld de, Bar
Bar:
    rrca
Foo:
    nop
```

On the other hand, *both* of those just omit the `ret` fragment. This is the ROM, no `C9` byte:

```
21 07 00 11 06 00 0F 00
```

Reply:
That looks like a bug, then.

Reply:
Since it's a bug independent of this new feature, I'll raise it separately.